### DISCUSSION TITLE
[Bug]: Submitting images to Frigate+ silently fails if clean_copy is disabled for snapshots

### DISCUSSION THREAD

**Discussion by SeanPM5** [blakeblackshear/frigate#13227] [Report a Bug] [answered]
### Describe the problem you are having

I updated my Frigate config yesterday to streamline things before moving over to a new mini PC, and noticed that my images were no longer submitting to Frigate+ and would instead silently fail to upload. 

Basically after I pressed the green "This is a person" button it'd disappear from the Frigate add-on UI like normal, the only problem is it wasn't actually making it over to the Frigate+ website. The image that disappeared would re-appear after a refresh (indicating it wasn't uploaded).

After investigating, it seems that it was caused by me turning off `clean_copy:` option under snapshots config. I tried turning that off because my snapshot settings were essentially "clean" already (no timestamps or bounding boxes, 100% quality) and figured I'd save a bit of storage space by having only one clean copy instead of two.

It makes sense that Frigate+ would require clean unobstructed PNG's for annotation and training purposes, but since I redid my entire config and it was failing silently, it took a while to troubleshoot and figure out what the issue was. It'd be nice if there was some kind of warning message on either the Frigate+ page or submission dialog when the user has `clean_copy: False`.

### Steps to reproduce

1. Disable clean copy in snapshots config (pasted below)
2. Restart Frigate
3. Wait for a new event/snapshot to appear on the Frigate+ page, and try to submit it
4. It'll disappear as if it uploaded, but it doesn't actually upload and fails silently. If you refresh the page it'll appear again.

### Version

0.14.0-da913d8

### Frigate config file

```yaml
snapshots:
  enabled: True
  bounding_box: False
  crop: False
  timestamp: False
  clean_copy: False
  retain:
    default: 14
    objects:
      person: 14
      car: 7
  quality: 100
```

### Relevant log output

```shell
N/A
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment, marked as answer]
Thanks, this has been fixed already for the next version
